pkg/app/app_test.go

```diff
@@ -217,6 +218,34 @@ func TestDeleteHostIsHandled(t *testing.T) {
 	}
 }
 
+func TestInfraConfigRequest(t *testing.T) {
+	controller := NewApp(&testInstanceManager{}, &testAccountManager{}, nil, nil, nil, "", nil, config.WebRTCConfig{STUNServers: []string{"foo.com:12345"}}, &config.Config{})
+	ts := httptest.NewServer(controller.Handler())
+	defer ts.Close()
+
+	res, _ := http.Get(ts.URL + "/v1/zones/foo/hosts/bar/infra_config")
+
+	expectedStatus := http.StatusOK
+	if res.StatusCode != expectedStatus {
```

Comment:
Use `cmp.Diff(http.StatusOK, res.StatusCode)`, and avoid having `expectedStatus` variable.